docs/tested_devices.md

```diff
@@ -11,5 +11,12 @@
 | MSI Prestige 14 Evo           | 14C4EMS1.120 04/25/2022 | ✔ Mode<br> ✔ Battery Limit<br> ✔ Cooler Boost | ✔ Keyboard Backlit (auto turn off not working)<br> ✔ FN -> Super<br> ❌ Webcam (greyed out)<br> ❌ USB Power Share (greyed out) |
 | MSI Summit E13 Flip Evo A11MT | 13P2EMS1.110 09/08/2021 | ✔ Mode<br> ✔ Battery Limit<br> ✔ Cooler Boost | ✔ Keyboard Backlit<br> ✔ FN -> Super<br> ✔ Webcam <br> ❌ USB Power Share (greyed out)                                         |
 | MSI GF63 Thin 11UC-866IN      | 16R6EMS1.104 11/14/2021 | ✔ Mode<br> ✔ Battery Limit<br> ✔ Cooler Boost | ✔ Keyboard Backlit (auto turn off not working)<br> ✔ FN -> Super<br> ✔ Webcam<br> ❌ USB Power Share (greyed out)              |
+| MSI PS63 Modern 8RC           | E16S1IMS.106 12/24/2018 | ❌ Mode (greyed out)<br> ✔ Battery Limit<br> ✔ Cooler Boost<br> ❌ Fan Control (missing) | ❌ Keyboard Backlit (greyed out)<br> ✔ FN -> Super<br> ✔ Webcam<br> ❌ USB Power Share (greyed out)              |
 
 If the table does not contain the device you are using, then you can add it.
+
+To get the EC Version and release date, you can use the following commands:
+```sh
+sudo dmidecode -s bios-version
+sudo dmidecode -s bios-release-date
```

Comment:
The EC version may differ from the BIOS version. It is better to take data from the application

Reply:
ok